frotz.py

```diff
@@ -235,61 +283,63 @@ def _configure_vte(self):
         else:
             bg_color = '#FFFFFF'
             conf.set('terminal', 'bg_color', bg_color)
-        self.set_colors(gtk.gdk.color_parse (fg_color),
-                            gtk.gdk.color_parse (bg_color),
-                            [])
-                            
+
+        self.set_colors(self.color_parse(bg_color),
+                        self.color_parse(fg_color), [])
+
         if conf.has_option('terminal', 'cursor_blink'):
             blink = conf.getboolean('terminal', 'cursor_blink')
         else:
-            blink = False
+            blink = 'False'
             conf.set('terminal', 'cursor_blink', blink)
-        
-        self.set_cursor_blinks(blink)
+        blink = Vte.CursorBlinkMode(0) if blink == 'False' else Vte.CursorBlinkMode(1)
+        self.set_cursor_blink_mode(blink)
 
         if conf.has_option('terminal', 'bell'):
             bell = conf.getboolean('terminal', 'bell')
         else:
-            bell = False
+            bell = 'False'
             conf.set('terminal', 'bell', bell)
         self.set_audible_bell(bell)
-        
+
         if conf.has_option('terminal', 'scrollback_lines'):
             scrollback_lines = conf.getint('terminal', 'scrollback_lines')
         else:
-            scrollback_lines = 1000
+            scrollback_lines = '1000'
             conf.set('terminal', 'scrollback_lines', scrollback_lines)
-            
-        self.set_scrollback_lines(scrollback_lines)
+
+        self.set_scrollback_lines(int(scrollback_lines))
         self.set_allow_bold(True)
-        
+
         if conf.has_option('terminal', 'scroll_on_keystroke'):
             scroll_key = conf.getboolean('terminal', 'scroll_on_keystroke')
         else:
-            scroll_key = False
+            scroll_key = 'False'
```

Comment:
Should be a boolean.  Reference the Terminal activity and PyGObject docs for the function.

Reply:
Because , configparser accepts <class 'str'> as *args.

Reply:
See https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.set

Reply:
I've seen no such problem on Terminal.